try.html

```diff
@@ -1149,6 +1153,9 @@ <h2 id="like-this"> Like This? </h2>
     to add Shields as a GitHub application on your GitHub account.
     </p>
   </div>
+  <div id="jira-sprint-completion">
+    <p>To get the <pre>Sprint ID</pre>, go to your Backlog view in your project, right click on your sprint name and get the value of <pre>data-sprint-id</pre>.</p>
```

Comment:
Could you use `<code>` instead of `<pre>`, please? Otherwise, it is rendered as a block, which breaks the sentence in a surprising and poorly readable way.